.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,35 @@
+name: Ansible Lint
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  ansible-lint:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: centos-stream-8
+            shortcut: cs8
+            container-name: el8stream
+          - name: centos-stream-9
+            shortcut: cs9
+            container-name: el9stream
+
+    name: ${{ matrix.name }}
+
+    container:
+      image: quay.io/ovirt/buildcontainer:${{ matrix.container-name }}
+
+    steps:
```

Comment:
Just leave below in steps:

```
 - name: Empty step
   run: echo "Success"
```

And remove all others steps, they should be part of the big PR introducing ansible-lint